<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SessionProperties</class>
 <widget class="QWidget" name="SessionProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>467</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="horizontalSpacing">
      <number>8</number>
     </property>
     <property name="verticalSpacing">
      <number>2</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="labelRenderer">
       <property name="text">
        <string>Renderer</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="DataComboBox" name="renderer"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelShaderCompiler">
       <property name="text">
        <string>Shader Compiler</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="DataComboBox" name="shaderCompiler"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="name"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelName">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="shaderCompilerOptions">
     <property name="title">
      <string>Shader Compiler Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="labelSprivVersion">
          <property name="text">
           <string>SPIR-V Version</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="DataComboBox" name="spirvVersion"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="autoMapBindings">
        <property name="text">
         <string>Automap Bindings</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="autoMapLocations">
        <property name="text">
         <string>Automap Locations</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="autoSampledTextures">
        <property name="text">
         <string>Auto Sampled Textures</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="vulkanRulesRelaxed">
        <property name="text">
         <string>Vulkan Rules Relaxed</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QLabel" name="labelShaderPreamble">
       <property name="text">
        <string>Shader Preamble</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ExpressionEditor" name="shaderPreamble">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="lineWrapMode">
        <enum>QPlainTextEdit::NoWrap</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelShaderIncludePaths">
       <property name="text">
        <string>Shader Include Paths (one per line)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ExpressionEditor" name="shaderIncludePaths">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="lineWrapMode">
        <enum>QPlainTextEdit::NoWrap</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DataComboBox</class>
   <extends>QComboBox</extends>
   <header>session/DataComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>ExpressionEditor</class>
   <extends>QPlainTextEdit</extends>
   <header>session/ExpressionEditor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
